CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
 ## 9.1.0
 This beta release includes a feature with a known issue: the `within` expression evaluates point features inconsistently across zoom levels if the point lies near the boundary of a GeoJSON object ([#16301](https://github.com/mapbox/mapbox-gl-native/issues/16301])).
 
+### Snapshotter
```

Comment:
Ah, sorry you moved faster than I could reply to your comment! I think we can delete this section and use the combined entry I suggested below, unless you have another suggestion @Chaoba